addons/mail/models/mail_alias.py

```diff
@@ -64,10 +64,10 @@ class Alias(models.Model):
         ('partners', 'Authenticated Partners'),
         ('followers', 'Followers only')], default='everyone',
         string='Alias Contact Security', required=True,
-        help="Policy to post a message on the document using the mailgateway.\n"
-             "- everyone: everyone can post\n"
-             "- partners: only authenticated partners\n"
-             "- followers: only followers of the related document or members of following channels\n")
+        help="Filters who can send emails this to email alias to create leads/opportunities : \n"
```

Comment:
Are you telling the generic alias mechanism is used only for leads / opportunities :D ?

Reply:
Obviously ! haha
I copied the spec as is. I'm a poor no-brainer !